### ISSUE TITLE
Particles dont show on linux

### ISSUE THREAD

**Issue by KernelDash** [codemaker4/raylib-particle-toy#1]
I compiled the toy with "g++ main.cpp -lraylib -lGL -lm -lpthread -ldl -lrt -lX11 -o main" but there are no particles. its just the window with a fps counter and nothing else

**Comment by codemaker4** [comment]
Made this a while ago in win10. I am now using linux and also have this issue.
I had a quick look around, and couldn't find a cause. I did find that I defined functions in .h files, which is pretty representative of my c/c++ skills at the time.
I really appreciate that you take the effort to open an issue for this, but unfortunately I currently do not have the time and motivation to debug this. But because this repo does get some attention for some reason, if someone wants to make a pull request fixing this, that would be highly appreciated.

**Comment by KernelDash** [comment]
Ok. Thanks for replying, if i have time i will try to fix it and let you know.

**Comment by KernelDash** [comment]
This toy is really cool. For anyone that wants to run it at this time you can run the exe with wine, it perfectly works like that.

**Comment by KernelDash** [comment]
Oh, now i realized that the particles come to a position above the mouse cursor but it may be because im on hyprland.

**Comment by lnc3l0t** [comment]
It doesn't work on X11 neither @LAZAR-code I think I'll check out the source too

**Comment by lnc3l0t** [comment]
The problem lies in the `GetRandomValue` calls in the Particle constructor. `SetRandomSeed(int)` must be called before the particles are initialized for example:
```c
SetRandomSeed(0);
for (long int i = 0; i < particleCount; i++)
{
    particles[i] = Particle(screenWidth, screenHeight);
}
```
https://github.com/codemaker4/raylib-particle-toy/blob/d0947764f7f2df5e4b4e6361fa5fffc630ca6f8b/main.cpp#L15-L19

**Comment by kirisakiken** [comment]
@lnc3l0t correct. Addressed here: https://github.com/codemaker4/raylib-particle-toy/pull/2

**Comment by codemaker4** [comment]
Fixed by pull request #2. Thanks everyone!

**Comment by KernelDash** [comment]
Thanks guys! I can confirm that it works!
